### PR TITLE
Phase2

### PR DESCRIPTION
**Pull request by AnasHagras**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+//
```
Comment (shereen-fathy) [inline index.html]: remove

File: script.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
+const courseContainer = document.querySelector(".courses-container");
+const searchButton = document.querySelector(".search-btn");
+let courses = [];
+searchButton.addEventListener("click", function(e) {
+    e.preventDefault();
+    updateTable();
+});
```
Comment (shereen-fathy) [inline script.js]: having a search button while you are doing the search on `keyup` is confusing for me so you have to depend on one of them not both

File: script.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+const searchText = document.querySelector(".search-input");
+searchText.addEventListener("keyup", function(e) {
+    updateTable();
+});
+
+function updateTable() {
+    let here = [];
+    here = filter(searchText.value);
```
Comment (shereen-fathy) [inline script.js]: you can do `const here=filter(searchText.value);` in one line right? also what is `here` used for? could you give it meaningful num?

File: script.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+fetch('https://api.npoint.io/74d742bda6f905ced3c7')
+    .then((response) => {
+        return response.json();
+    }).then((arr) => {
+        courses = arr;
+        for (let i = 0; i < arr.length; i++) {
+            render(arr[i]);
+        }
+    });
```
Comment (shereen-fathy) [inline script.js]: you should use your json-server here,  right?

File: script.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    startsSpan.appendChild(icon1);
+    startsSpan.appendChild(icon2);
+    startsSpan.appendChild(icon3);
+    startsSpan.appendChild(icon4);
+    startsSpan.appendChild(icon5);
+    icon1.classList.add("fa-solid", "fa-star");
+    icon2.classList.add("fa-solid", "fa-star");
+    icon3.classList.add("fa-solid", "fa-star");
+    icon4.classList.add("fa-solid", "fa-star");
+    icon5.classList.add("fa-regular", "fa-star-half-stroke");
```
Comment (shereen-fathy) [inline script.js]: you should add stars here based on the ratings, right?

**Review by shereen-fathy — CHANGES_REQUESTED** [review]
please work on fixing the comments

File: script.js (hunk 5 above)
Comment (AnasHagras) [reply script.js]: no , mr ahmed ghannam said that we have to do it just style

File: script.js (hunk 4 above)
Comment (AnasHagras) [reply script.js]: yes json-server

File: script.js (hunk 3 above)
Comment (AnasHagras) [reply script.js]: yeah it can be in one line , here means the current courses after filtering , i will rename it okay

File: script.js (hunk 2 above)
Comment (AnasHagras) [reply script.js]: i was testing the both , but i forgot to delete one of them

File: index.html (hunk 1 above)
Comment (AnasHagras) [reply index.html]: okay , by wrong :"D

**Review by shereen-fathy — APPROVED** [review]
thank you for addressing pr comment, and thank you for the deployment

**Pull request merged**